src/lib/utils.js

```diff
@@ -412,3 +416,31 @@ function parseAroundLatLngFromString(value) {
     lng: parseFloat(pattern[2]),
   };
 }
+
+function enhanceConfiguration(searchParametersFromUrl) {
```

Comment:
I'd add some tests. This a good opportunity since it's now standalone.

Reply:
Not the scope of the PR as well, let's try to keep them small 👍

Reply:
I do not agree. You should test this function.

Reply:
One way to take a decision on this is to ask ourselves "Is this code already tested by other tests". i.e. if I completely break this module, will tests still pass? If yes then this is an issue and we need to add tests on it. If not, less of an issue but we'd have to discuss about which parts we consider needed for unit tests or not.

As a general rule we can agree that, when moving code around for your own needs, if this code is not tested (our fault), let's add tests to it. Especially since it is not a big function and especially if you are moving this to be reused in a new feature you are developing.

On a side note, this code is probably mine so I'd be happy to write some tests with you @iam4x in pair, deal? Book some time with me!

Parts of the code seems to be tested, maybe not the function itself but the way it's used by others. So most of the tests should already be written: https://github.com/algolia/instantsearch.js/commit/0ea3bef85922b876b4efacf64f3025fb730ee982

Reply:
Yes this code need to be tested but does it goes with the feature of this PR?

We had a meeting the day I wrote those comments with the team about having too much big PRs that are hard to review and here I'm trying to apply what we discussed.

I totally agree on the fact that we may have identified a flaw of our code because of the lack of unit testing of this function. But I really think it need to be done in another PR with an another issue linked, because here we are raising the issue.

That being said, I see this can be discussable in this case since the PR is not that big yet.

Reply:
Let's start to do stack PRs! I'll drop a PR with the tests, this should fit with everybody?